attachment_base_synchronize/models/attachment.py

```diff
@@ -56,8 +57,7 @@ def _compute_hash(self):
     @api.model
     def run_attachment_metadata_scheduler(self, domain=None):
         if domain is None:
-            domain = []
-        domain.append(('state', '=', 'pending'))
+            domain = [('state', '=', 'pending')]
```

Comment:
I agree with @florian-dacosta on this one. An alternative to changing the behavior is to use a context override to stop the hard coded domain, but IMO explicitly adding is better than explicitly removing.